pkg/gadget-collection/gadget-collection.go

```diff
@@ -75,7 +75,6 @@ func TraceFactoriesForLocalGadget() map[string]gadgets.TraceFactory {
 	return map[string]gadgets.TraceFactory{
 		"audit-seccomp":     auditseccomp.NewFactory(),
 		"capabilities":      capabilities.NewFactory(),
-		"dns":               dns.NewFactory(),
```

Comment:
I think we should. I create an issue to discuss it: https://github.com/inspektor-gadget/inspektor-gadget/issues/1125

Reply:
> Why removing it? Do you plan to remove local-gadget interactive?

Yes. I don't see any value in keeping both modes around. We should focus on a single mode and improve it as much as possible instead of spreading our efforts into different modes.